courses/views.py

```
@@ -0,0 +1,122 @@
+    print(teacher_classes)
+
+    context = {
+        "courses": courses,
+        "enrolled_classes": enrolled_classes,
+        'teacher_classes': teacher_classes,
+        "role": role,
+    }
+    return render(request, "courses/course_list.html", context)
+
+
+class CreateCourseView(LoginRequiredMixin, CreateView):
```

Ruff (0.8.2)

Undefined name `LoginRequiredMixin`

(F821)